src/app/page.tsx

```diff
@@ -1,4 +1,5 @@
 import {LikeButton} from './components/LikeButton';
```

Comment:
ここはむしろスペースあった方が良いかな！
インラインで書かれているものはスペースをなくすってイメージで書いてます
```suggestion
import { LikeButton } from './components/LikeButton';
```